$NetBSD: patch-src_gl-system-install,v 1.1.1.1 2011/11/20 19:11:49 ryoon Exp $

* Do not check gl-setup in $PATH, it is useless with staged installation.

--- src/gl-system-install.orig	2011-11-20 12:03:57.000000000 +0000
+++ src/gl-system-install
@@ -92,24 +92,24 @@ cp -R hooks/* $buildroot$gl_hooks_dir ||
 # ----
 
 # check if $gl_bin_dir is in $PATH and advise the user if needed
-which=`which gl-setup 2>/dev/null`
-
-path_advice="
-Since gl-setup MUST be run from the PATH (and not as src/gl-setup or such),
-you must fix this before running gl-setup.  The simplest way is to add
-
-    PATH=$gl_bin_dir:\$PATH
-
-to the end of your bashrc or similar file.  You can even simply run that
-command manually each time you log in and want to run a gitolite command."
-
-[ -z "$which" ] && die "                ***** WARNING *****
-gl-setup is not in your \$PATH.
-$path_advice"
-
-which=`dirname $which`
-[ "$which" != "$gl_bin_dir" ] && die "                ***** WARNING *****
-$which precedes $gl_bin_dir in your \$PATH,
-and it *also* contains gl-setup.  This is almost certainly going to confuse
-you or me later.
-$path_advice"
+#which=`which gl-setup 2>/dev/null`
+#
+#path_advice="
+#Since gl-setup MUST be run from the PATH (and not as src/gl-setup or such),
+#you must fix this before running gl-setup.  The simplest way is to add
+#
+#    PATH=$gl_bin_dir:\$PATH
+#
+#to the end of your bashrc or similar file.  You can even simply run that
+#command manually each time you log in and want to run a gitolite command."
+#
+#[ -z "$which" ] && die "                ***** WARNING *****
+#gl-setup is not in your \$PATH.
+#$path_advice"
+#
+#which=`dirname $which`
+#[ "$which" != "$gl_bin_dir" ] && die "                ***** WARNING *****
+#$which precedes $gl_bin_dir in your \$PATH,
+#and it *also* contains gl-setup.  This is almost certainly going to confuse
+#you or me later.
+#$path_advice"
